setup.py

```diff
@@ -46,8 +46,6 @@
         'License :: OSI Approved :: Apache Software License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```

Comment:
Looks like I forgot to add py3.7 here when I added it to the test matrix, would you add it?

Reply:
3.7 is included, both here: 
https://github.com/Princeton-CDH/django-pucas/blob/feature/drop-python-2/setup.py#L52

and in the test matrix:
https://github.com/Princeton-CDH/django-pucas/blob/feature/drop-python-2/.travis.yml#L8

Do you mean we should extend to Python 3.8? Let me know if I'm missing something

Reply:
oh! I just thought it was missing because I didn't expand the diff, my mistake. No need to expand to 3.8 right now

Reply:
Oh! yeah absolutely: #8